src/main/java/de/tum/p2p/onion/forwarding/netty/NettyOnionForwarder.java

```diff
@@ -128,7 +128,7 @@ private NettyOnionForwarder(Builder builder) {
             log.info("Onion Server Channel has been initialized on {}", serverChannel.localAddress());
 
         } catch (InterruptedException | ExecutionException | TimeoutException e) {
-            throw new OnionInitializationException("Failed to initialize onion server channel");
```

Comment:
please revert those channel -> setup, authentication -> auth changes. probably done accidentally.
just override these my onion forwarding classes with version from master brach (being on you feature branch of course):
`git checkout origin/master <filepath>`
e.g. `git checkout origin/master src/main/java/de/tum/p2p/onion/forwarding/netty/NettyOnionForwarder.java`